x-pack/plugins/ingest_manager/common/types/models/agent.ts

```diff
@@ -16,15 +16,21 @@ export type AgentStatus = 'offline' | 'error' | 'online' | 'inactive' | 'warning
 
 export interface NewAgentAction {
   type: 'CONFIG_CHANGE' | 'DATA_DUMP' | 'RESUME' | 'PAUSE';
-  data?: string;
+  data?: any;
```

Comment:
Is it possible to lock the type down?

Reply:
We currently do not have the type here, I created a issue so we can define a schema and enforce it https://github.com/elastic/kibana/issues/62440